netbox/templates/tenancy/object_contacts.html

```diff
@@ -0,0 +1,27 @@
+{% extends 'generic/object.html' %}
```

Comment:
We probably want to reference the default object template at the base template, as we do for the object journal and changelog views. This ensures that any customizations to the page header remain intact across tabs. For instance, if we use the generic `object.html` base template, we lose the breadcrumbs under the contacts tab:

![breadcrumbs1](https://github.com/netbox-community/netbox/assets/13487278/6f59cd0f-ff52-4c52-bf91-00f328db7ea4)

---

![breadcrumbs2](https://github.com/netbox-community/netbox/assets/13487278/7b20841f-8961-44b6-91a3-be04b1063780)

IMO it's fine to replicate what we do for the other views:

```python
        # Default to using "<app>/<model>.html" as the template, if it exists. Otherwise,
        # fall back to using base.html.
        if self.base_template is None:
            self.base_template = f"{model._meta.app_label}/{model._meta.model_name}.html"
```